dispatcher/tcp/tcp.go

```diff
@@ -108,14 +108,14 @@ func (d *TCP) handleConn(conn net.Conn) error {
 	// dial and relay
 	rc, err := net.Dial("tcp", server.Target)
 	if err != nil {
-		return fmt.Errorf("[tcp] handleConn dial error: %w", err)
+		return fmt.Errorf("[tcp] %s <-> %s <-x-> %s handleConn dial error: %w", conn.RemoteAddr(), conn.LocalAddr(), rc.RemoteAddr(), err)
```

Comment:
Can the value of conn.RemoteAddr() (type is net.Addr) be printed with flag %s? I'm not sure.

Reply:
Maybe %v is a safer solution

Reply:
It works. And you used it too at L121.

Reply:
LGTM. I found that the previous code used the flag %s.